include/nbl/builtin/hlsl/ieee754.hlsl

```diff
@@ -142,13 +142,47 @@ NBL_CONSTEXPR_INLINE_FUNC FloatingPoint copySign(FloatingPoint to, FloatingPoint
 	return bit_cast<FloatingPoint>(toAsUint | extractSignPreserveBitPattern(from));
 }
 
-template <typename FloatingPoint NBL_FUNC_REQUIRES(concepts::FloatingPointLikeScalar<FloatingPoint>)
-NBL_CONSTEXPR_INLINE_FUNC FloatingPoint flipSign(FloatingPoint val, bool flip = true)
+namespace impl
+{
+template <typename T NBL_STRUCT_CONSTRAINABLE>
```

Comment:
you need anoither template parameter `U` for the bool, for your vectorized variant